articles/quickstart/native/android/07-authorization.md

```diff
@@ -67,9 +73,9 @@ if (roles.contains("admin")) {
 ```
 
 ::: note
-Notice that you'll find the `roles` information within the `appMetadata` HashMap and not in the `userMetadata`. Application metadata cannot be modified by users, whereas User metadata can be.
+Application metadata cannot be modified by users but the user metadata can be modified. Because of that, the information about user roles is in the `appMetadata` HashMap, not in the `userMetadata` HashMap.
```

Comment:
I would use:
"Application metadata cannot be modified by users but the user metadata can. Because of that, information about user roles is stored in the `appMetadata` Map"